app/views/includes/page-title.html

```diff
@@ -0,0 +1,3 @@
+{% if subTitle %}{{ subTitle }} - {% endif %}
+{% if title %}{{ title }}{% endif %} 
+- WMT
```

Comment:
We not including any detail about the tab they are on, e.g. `National - HMPPS - Capacity`?

Reply:
Andy answered this F2F